### ISSUE TITLE
Hyperparameters for SFT?

### ISSUE THREAD

**Issue by peterjc123** [jiaweizzhao/GaLore#15]
Thanks for the great work. One thing I'm curious about is that does it actually work well on SFT for LLMs? It is not covered in the paper, as well. I tried the following parameters on a 2B-sized model, but it leads to very slow convergence. Could you please give me some advice?
```yaml
lr: 5e-5
galore_rank: 64
galore_update_proj_gap: 200
scale: 0.25
proj_type: std
```

**Comment by peterjc123** [comment]
After some trials, I am able to find a pair of hyper-parameters that performs on par with LoRA/DoRA with regard to training loss (with `lora_rank=64, lora_alpha=16, lora_dropout=0.5`).
```yaml
lr: 3e-5
galore_rank: 256 
galore_update_proj_gap: 200
galore_scale: 4.0
galore_proj_type: std
weight_decay: 1e-4
lr_schedule: cosine
```

**Comment by pixeli99** [comment]
mark
